sass/_cover.scss

```diff
@@ -1,138 +1,134 @@
 .cover {
 	background-color: $background-color-cover;
-    color: #fff;
-    overflow: hidden;
-    padding: 156px 0 80px;
-    position: relative;
+	color: #fff;
+	overflow: hidden;
+	padding: 156px 0 80px;
+	position: relative;
 	transition: opacity .25s;
 	z-index: $z-index-cover;
-	
+
 	&-header {
 		@include site-width;
 		position: relative;
-		
-        p {
-            color: #fff;
-        }
-        
+
+		p {
+			color: #fff;
+		}
+
 		a {
-            color: #fff;
-            
-            &:hover {
-                border-color: #fff;
-            }
-            
-            &.author:hover {
-                border-color: transparent;
-                
-                span {
-                    border-color: #fff;
-                }
-            }
-        }
-        
-        .taxonomy-description {
+			color: #fff;
+
+			&:hover {
+				border-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.